debian/control

```diff
@@ -0,0 +1,20 @@
+Source: qubes-rpm-oxide
+Maintainer: Demi Marie Obenour <demi@invisiblethingslab.com>
+Section: misc
+Priority: optional
+Standards-Version: 5.0.0
```

Comment:
It seems the most recent policy version is 4.6.0.1 (https://www.debian.org/doc/debian-policy/index.html)